napalm_logs/base.py

```diff
@@ -171,9 +171,14 @@ def start_engine(self):
         Start the child processes (one per device OS),
         open the socket to start receiving messages.
         '''
-        # TODO prepare the binding to be able to listen to syslog messages
-        skt = None
-        # TODO
+        skt = socket.socket(socket.AF_INET,
```

Comment:
Please check if hostname/ip is IPv6; if so, the socket type should be `AF_INET6`